### ISSUE TITLE
Create block for image thumbnail slider

### ISSUE THREAD

**Issue by pardeepgera23** [hlxsites/aldevron#114]
**Summary**

Create a block for the image thumbnail slider for showing gallery.
Example link from current website: 
https://www.aldevron.com/campus

**Comment by pardeepgera23** [comment]
@Kummu007 - Please validate

**Comment by Kummu007** [comment]
Added a bug.

Image Thumbnail Slider- Alt Text and Tooltip are Missing #135

**Comment by pardeepgera23** [comment]
@Kummu007 - Please validate

**Comment by Kummu007** [comment]
Added

 Image Thumbnail Slider- Mobile View issue #177
Image Thumbnail Slider- Image overriding #178

**Comment by Kummu007** [comment]
This is working as expected.
